job-descriptions/software-engineer.md

```diff
@@ -1,48 +1,49 @@
 ![logo](https://sourcegraph.com/.assets/img/sourcegraph-light-head-logo.svg)
 
-# Software Engineer (San Francisco or remote)
+# Software Engineer

```

Comment:
The fact that we are distributed-friendly is now a bit hard to find in here

Reply:
I'll revert this